### PR TITLE
Fixed abandoned trigger last checks

### PR DESCRIPTION
**Pull request by zhelyabuzhsky**
In case of an error during deletion of trigger, its last check were left unremoved. This led not only to clogging the database, but also to problems when creating a trigger with the same identifier.

In this PR:
- added command to clean up the database from abandoned lastchecks
- the section of the code has been rewritten so that the removal of the trigger takes place in one transaction with the removal of its last check

### CODE REVIEW

**Review by dmitryanchikov — COMMENTED** [review]
Overall looks good, I might miss but are there any new tests for checking of last check removing on trigger removing?

File: database/redis/last_check.go [hunk 1]
```diff
@@ -71,14 +75,20 @@ func (connector *DbConnector) getSelfStateCheckCountKey(isRemote bool) string {
 	return selfStateChecksCounterKey
 }
 
+func appendRemoveTriggerLastCheckToRedisPipeline(context context.Context, pipe redis.Pipeliner, triggerID string) redis.Pipeliner {
```
Comment (kissken) [inline database/redis/last_check.go:78]: optional: Variable name is same as package name. Maybe better change to "ctx"?

**Review by kissken — COMMENTED** [review]
Left a small cosmetic comment.

File: database/redis/last_check.go (hunk 1 above)
Comment (zhelyabuzhsky) [reply database/redis/last_check.go]: Changed to `ctx`

**Comment by zhelyabuzhsky** [comment]
> are there any new tests for checking of last check removing on trigger removing?

[Added](https://github.com/moira-alert/moira/pull/753/files#diff-ecefb56af45b7a8512d30dba0fcbb400176762b1464abaa97f518d4fed393536R288-R291) assertion to test case, thanks )

**Review by dmitryanchikov — APPROVED** [review]

**Review by kissken — APPROVED** [review]

**Pull request merged**